input.html

```diff
@@ -4,9 +4,35 @@
     <meta charset="UTF-8" />
     <meta http-equiv="X-UA-Compatible" content="IE=edge" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <title>Document</title>
+    <link rel="stylesheet" href="style.css" />
+    <title>Input Page</title>
   </head>
   <body>
-    <header><h1>input page</h1></header>
+    <header><h1>Create</h1></header>
+
+    <main>
+      <section>
+        <h2>Add question:</h2>
+      </section>
+
+      <section>
+        <h2>Add answer:</h2>
+      </section>
+
+      <section>
+        <h2>Add tags (divided by comma):</h2>
+      </section>
+
+      <a href="">Button "Submit"</a>
```

Comment:
You should rather use a `form` element  with inputs and labels for this (we are going to speak about this tomorrow, but [you can have a look here](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/form)).